python/python/glide/async_commands/redis_modules/json/json.py

```diff
@@ -0,0 +1,104 @@
+# Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
```

Comment:
Lets remove the json folder, we can always add it later on if the json module will need to be extended

Reply:
General note: please add json to the __all__ in glide/__init__